### ISSUE TITLE
(git) Bundled git-lfs version is different from the latest git-lfs version available

### ISSUE THREAD

**Issue by ba-tno** [chocolatey-community/chocolatey-coreteampackages#1723]
## Expected Behavior

When installing git from chocolatey without any additional options (`choco install git`) git-lfs is installed as well.
However, it appears that an update to git-lfs does not propagate to the chocolatey git package.

At the current moment there is a separate [git-lfs](https://community.chocolatey.org/packages/git-lfs) package with version `3.0.1`. However, the git-lfs version bundled with the chocolatey git package is `2.13.3`:

```ps
PS C:\Program Files\Git\mingw64\bin> git-lfs --version
git-lfs/2.13.3 (GitHub; windows amd64; go 1.16.2; git a5e65851)
```

Running `choco outdated` does not mark git as outdated, while git-lfs (an underlying package) is actually outdated.

My expectation is that when git-lfs is updated the git chocolatey package also uses the new version and is marked as outdated. Then, when I upgrade the git choco package (`choco upgrade git`) the newer git-lfs version would also be picked up.

## Current Behavior

The bundled git-lfs version which is installed by default is older than the latest available git-lfs version.

## Possible Solution

It would be nice if the git choco package would somehow make use of the git-lfs choco package. In that way, an update of git-lfs could propagate to the git package.

## Steps to Reproduce (for bugs)

1. Install git without any options: `choco install git`
2. Check the git-lfs version: `git-lfs --version`
3. Compare it to the latest version of git-lfs on chocolatey: `choco list git-lfs`

## Your Environment

* Package Version used: 2.33.0.2
* Operating System and version: Windows 10 version 1909 (OS Build 18363.1801)
* Chocolatey version: 0.11.2
* Install/uninstall gist:

**Comment by pascalberger** [comment]
The `git` package is a wrapper around [Git for Windows installer](https://gitforwindows.org/), which is a bundler for Git and different tools, including git-lfs. The `git-lfs` package is a wrapper around the [Git Large File Storage installer](https://git-lfs.github.com/). The `git` package doesn't have a dependency on the `git-lfs` package. For Chocolatey packages it is best practice to be just light wrappers around installers, which both packages currently are, instead of implementing additional logic themselves.

While git-lfs 3.0.1 recently has been released, and therefore `git-lfs` package been updated, there's no update to the Git for Windows installer yet available which bundles git-lfs 3.0.1. This is not something we can fix at the Chocolatey package level, but something which Git for Windows needs to update. As soon as there is a new version of Git for Windows available the `git` Chocolatey package will automatically be updated.

If you prefer to use a newer version of git-lfs than is shipped with latest version of Git for Windows, you're free to use the `git-lfs` Chocolatey package. Please note that in this case it is up to you to make sure that the different Git tools are compatible.

**Comment by ba-tno** [comment]
That's clear, thank you for the explanation. I was not aware that this was a wrapper of the official installer.
As there are switches to toggle git-lfs I thought this was something which was added in the chocolatey package. If the official Git for Windows installer is (currently) bundled with an older version of git-lfs I agree that the chocolatey package should have the same behaviour.
